test/com/moclojer/resources/moclojer.yml

```
@@ -117,3 +117,13 @@
         }
 
+# this is a websocket route
+- websocket:
+    path: /ws/echo
+    on-connect:
+      response: '{"status": "connected", "message": "Welcome to WebSocket Echo!"}'
+    on-message:
+      - pattern: "ping"
+        response: "pong"
+      - pattern: '{"echo": "{{json-params.echo}}"}'
+        response: '{"echoed": "{{json-params.echo}}"}'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)